src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -546,7 +546,28 @@ def __call__(self):
 
         self.fixupProcess()
 
-        # repack is only run at CERN and has a streaming format as input that should never use lazy-download
+        # for CERN:
+        #   enable lazy-download for fastCloning for all CMSSW_7_5 jobs (currently off)
+        #   enable lazy-download for all merge jobs at CERN
+        if socket.getfqdn().endswith("cern.ch"):
+
+            cmsswVersion = os.environ['CMSSW_VERSION']
+            if cmsswVersion.startswith("CMSSW_7_5") and False:
+                print "Using fastCloning/lazydownload"
+                self.process.add_(
+                    cms.Service("SiteLocalConfigService",
+                                overrideSourceCloneCacheHintDir=cms.untracked.string("lazy-download")
+                                )
+                    )
+            elif funcName == "merge":
```

Comment:
One sets lazy-download, the sets "if the job uses fast cloning -> use lazy-download"